rkt/run.go

```diff
@@ -62,15 +62,40 @@ End the image arguments with a lone "---" to resume argument parsing.`,
 	flagLocal       bool
 )
 
+//
+// Search a series of directories for a stage1 image
+//
+
+var stage1ImagePath = []string{
+	"./",
```

Comment:
If rkt cannot read /proc/self/exe, we fallback on the current directory. What is the rationale for this?

In normal conditions, /proc/self/exe is readable.  But if /proc is not mounted or if it was mounted in a pid namespace where the current rkt process is not visible, then /proc/self/exe is not readable.

I would prefer to skip the stage1ImagePath[0] entry in the for loop below when /proc/self/exe is not readable.